packages/hash/frontend/src/components/layout/PageHeader/NotificationsDropdown.tsx

```diff
@@ -44,6 +44,14 @@ export const NotificationsDropdown: React.FC = () => {
           "&:hover": {
             color: theme.palette.common.white,
             backgroundColor: theme.palette.blue["70"],
+            ...(hasNotifications && {
+              backgroundColor: theme.palette.blue[80],
+              boxShadow: "unset",
```

Comment:
When there are notifications and this is hovered, it doesn't have a shadow but has a darker background

<img width="195" alt="Screenshot 2022-03-23 at 03 53 02" src="https://user-images.githubusercontent.com/16928688/159614582-87a26faa-8629-464d-b803-01e04f6ed652.png">